src/test/java/Homework19.java

```diff
@@ -0,0 +1,26 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+
+    @Test
+    @Parameters ({"BaseURL", "HomeURL"})
+    public static void deletePlaylist () throws InterruptedException {
+        launchBrowser("https://bbb.testpro.io/");
```

Comment:
LaunchBrowser is annotated as @BeforeMethod, so it will be called automatically before a method with @Test annotation (You don't need to call it)